lib/components/Slider/Slider.tsx

```diff
@@ -74,13 +78,20 @@ export const Slider = ({ label, onValueChange, value }: SliderProps) => {
     setFocused(details.focusedIndex === 0);
   };
 
+  const getTooltipValue = () => {
+    if (rightLabel) {
+      return `${100 - value} - ${value} %`;
+    }
+    return `${value}%`;
```

Comment:
Very minor nitpick, but there is a space before % sign when rightLabel is used. Maybe there should be a space without it too?